scripts.js

```diff
@@ -0,0 +1,135 @@
+
+function loadSpeakers() { // eslint-disable-line no-unused-vars
+  featuredSpeakers.innerHTML = `
+        <div>
+          <h2>Featured Speakers</h2>
+          <div class="divline"></div>
+        </div>
+        <div class="speakers-container">
+          <div class="speakers" id="1">
+          <div class="speakerImg"><img src="${speakersDetails[0].picture}" alt=""></div>
+            <div class="speakersInfo">
+              <h3>${speakersDetails[0].name}</h3>
+              <h4>${speakersDetails[0].occupation}</h4>
+              <div></div>
+              <p>${speakersDetails[0].description}</p>
+            </div>
+          </div>
+          <div class="speakers" id="2">
+            <div class="speakerImg"><img src="${speakersDetails[1].picture}" alt=""></div>
+            <div class="speakersInfo">
+              <h3>${speakersDetails[1].name}</h3>
+              <h4>${speakersDetails[1].occupation}</h4>
+              <div></div>
+              <p>${speakersDetails[1].description}</p>
+            </div>
+          </div>
+          <div class="speakers hideSpeakers" id="3">
+            <div class="speakerImg"><img src="${speakersDetails[2].picture}" alt=""></div>
+            <div class="speakersInfo">
+              <h3>${speakersDetails[2].name}</h3>
+              <h4>${speakersDetails[2].occupation}</h4>
+              <div></div>
+              <p>${speakersDetails[2].description}</p>
+            </div>
+          </div>
+          <div class="speakers hideSpeakers" id="4">
+            <div class="speakerImg"><img src="${speakersDetails[3].picture}" alt=""></div>
+            <div class="speakersInfo">
+              <h3>${speakersDetails[3].name}</h3>
+              <h4>${speakersDetails[3].occupation}</h4>
+              <div></div>
+              <p>${speakersDetails[3].description}</p>
+            </div>
+          </div>
+          <div class="speakers hideSpeakers" id="5">
+            <div class="speakerImg"><img src="${speakersDetails[4].picture}" alt=""></div>
+            <div class="speakersInfo">
+              <h3>${speakersDetails[4].name}</h3>
+              <h4>${speakersDetails[4].occupation}</h4>
+              <div></div>
+              <p>${speakersDetails[4].description}</p>
+            </div>
+          </div>
+          <div class="speakers hideSpeakers" id="6">
+            <div class="speakerImg"><img src="${speakersDetails[5].picture}" alt=""></div>
+            <div class="speakersInfo">
+              <h3>${speakersDetails[5].name}</h3>
+              <h4>${speakersDetails[5].occupation}</h4>
+              <div></div>
+              <p>${speakersDetails[5].description}</p>
+            </div>
+          </div>
+        </div>
+        <button class="showMore" onclick="showSpeakers();">MORE ▼</button>`;
+}
+
+function showSpeakers() { // eslint-disable-line no-unused-vars
+  const showMore = document.querySelector('.showMore');
```

Comment:
- [OPTIONAL] Great job implementing the `show more` functionality in the speaker's section, I am recommending you add the `show less` button that will decreament the speaker's cards ^_^

![design](https://user-images.githubusercontent.com/80895497/164351536-d97730db-59d3-4b1c-a234-e51bac1c3283.PNG)